### PR TITLE
Add CustomVoiceApi Java sample

### PR DESCRIPTION
**Pull request by baike960**
I am a Microsoft intern and I wrote a CustomVoiceApi Java sample upon request from a colleague. The code has been tested and could run correctly.

### CODE REVIEW

File: CustomVoice-API-Samples/Java/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+## Build the samples
+
+1. First, you must obtain a standard (not free) Speech API subscription key by following instructions in Microsoft Cognitive Services subscription.
+1. To run the sample, make sure that you manually include all the jar files inside the lib folder to the project. 
+1. You need to pass in arguments of endpoint, ibizaStsUrl, subscriptionKey, localInputTextFile, locale, voiceName and concatenateResult to the VoiceSynthsisAPIToJAVA.java file based on the information of the region, your subscription key, your designated voice and your desirable format of output wav file.
+
+## Note:
+
+1.The input text file should be Unicode format with 'UTF-8-BOM' (you can check the text format with your preferred advanced text editor), like the one zh-CN.txt, and should be more than 50 lines.
+1.The voiceId should pick up from MS guys or from the function getVoiceId in the java file.
```
Comment (boltomli) [inline CustomVoice-API-Samples/Java/README.md]: Not sure what do you mean with this note. Who are MS guys? Please kindly elaborate.

File: CustomVoice-API-Samples/Java/src/VoiceSynthesisMain.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+		VoiceSynthsisAPIs(endpoint, ibizaStsUrl, subscriptionKey, localInputTextFile, locale, voiceName, concatenateResult);
+
+	}
+	
+	
+	private static void VoiceSynthsisAPIs(String endpoint, String ibizaStsUrl, String subscriptionKey, String
+			localInputTextFile, String locale, String voiceName, String concatenateResult) 
+					throws ApiException, IOException, InterruptedException, JSONException
+    {
+
```
Comment (boltomli) [inline CustomVoice-API-Samples/Java/src/VoiceSynthesisMain.java]: Please clean up the overall style of the source code files.

File: CustomVoice-API-Samples/Java/README.md (hunk 1 above)
Comment (baike960) [reply CustomVoice-API-Samples/Java/README.md]: I will make modifications on Monday. Thanks!

File: CustomVoice-API-Samples/Java/src/VoiceSynthesisMain.java (hunk 2 above)
Comment (baike960) [reply CustomVoice-API-Samples/Java/src/VoiceSynthesisMain.java]: As for the dependencies, they are organized as jar files located under the lib folder. The jar files are imported through the class path file, so they do not have to be manually imported.

**Comment by baike960** [comment]
Hi Song,
I re-organized the source files and added a pom file to use Maven to automatically download the required files for dependencies. I updated the README.md to record the steps. Please review the changes. Thanks.

**Comment by baike960** [comment]
The coding style was cleaned up

**Comment by boltomli** [comment]
@hepower Can you help review the change as well? Thanks.

**Comment by hepower** [comment]
it is ok for me. thanks.

**Pull request merged**